test/python/test_status_api.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+WEBSERVICE_POLICY = """
+- !policy
+  id: conjur/authn-oidc/okta
+  body:
+  - !webservice
+    id: status
+    annotations:
+      description: Status service to check that the authenticator is configured correctly
+
+  - !group
+    id: operators
+    annotations:
+      description: Group of users who can check the status of the authenticator
+
+  - !permit
+    role: !group operators
+    privilege: [ read ]
+    resource: !webservice status
+
+  - !webservice
+    annotations:
+      description: Authentication service for okta, based on OpenID Connect.
+
+  - !variable
+    id: provider-uri
+
+  - !variable
+    id: id-token-user-property
+
+  - !group
+    id: users
+    annotations:
+      description: Group of users who can authenticate using the authn-oidc/okta authenticator
+
+  - !permit
+    role: !group users
+    privilege: [ read, authenticate ]
+    resource: !webservice
+
+- !policy
+  id: conjur/authn-azure
+  body:
+  - !webservice
+    id: status
+    annotations:
+      description: Status service to check that the authenticator is configured correctly
```

Comment:
Missing tabs

Reply:
These objects aren't supposed to be included in the first policy. They are loaded into root along side the above policy